### PR TITLE
Add backup RPC

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

Adds a fallback URL incase the main RPC goes down  

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes, checking [accessibility](https://www.a11yproject.com/checklist/) etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver
- [x] Add release notes to [unreleased changelog](https://github.com/teamsempo/SempoBlockchain/blob/master/CHANGELOG.md#unreleased)

### CODE REVIEW

**Comment by michielderoos** [comment]
[Add backup RPC URL](https://trello.com/c/3HljzBt5/792-add-backup-rpc-url)

**Review by tristanhcole — COMMENTED** [review]
LGTM. Need to update `specific_config_template.ini` https://github.com/teamsempo/SempoBlockchain/blob/94da610227ba9345e45d90d3455c9acba3fe95c8/config_files/templates/specifc_config_template.ini#L36. Small nit

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [v1.8.2] - 2021-03-16
 
+### Add fallback URL incase the main RPC is experiencing downtime
```
[the remark this answers is not in the document]
Comment (michielderoos) [reply CHANGELOG.md]: Fixed @tristanhcole, Ready for 👀 !

**Review by tristanhcole — APPROVED** [review]
good to go!

**Pull request merged**